### PR TITLE
Bug/safety check find logic

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

N/A

### Change description ###

Bug fix for safety check for find

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts [hunk 1]
```diff
@@ -2,23 +2,55 @@ import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { provideMockStore } from '@ngrx/store/testing';
-
+import { of } from 'rxjs';
+import { UserService } from 'src/app/services/user/user.service';
```
Comment (connorpgpmcelroy) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts:8]: src should be taken out here

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Just one minor change needed

File: src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts]: @connorpgpmcelroy  this is coming from master merge. But, still fixed it.

**Review by kuda1992 — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
Comment addressed - LGTM

**Pull request merged**